### PR TITLE
first commit home work 12

### PR DESCRIPTION
**Pull request by Oleksii1985**

### CODE REVIEW

File: lesson_6/Array.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  } else {
+    newArrayFromUser.push(arrayValue);
+  }
+  lenghtArray -= 1;
+  console.log(newArrayFromUser);
+}
+console.log(newArrayFromUser);
+
+
+newArrayFromUser.sort();
```
Comment (web-scavenger) [inline lesson_6/Array.js:22]: This is a big problem that you do all updates with same array. Can you see actual results in every log message? Don't think so

Comment (web-scavenger) [inline lesson_6/Array.js:22]: default `sort()` method will sort data like a `string`. Check the result with array, for example `[1, 2, 3, 12, 24, 41]`

Comment (Oleksii1985) [reply lesson_6/Array.js:22]: done

Comment (Oleksii1985) [reply lesson_6/Array.js:22]: ok

**Pull request merged**